bin/e2e-test-client/src/tests/transfers.rs

```diff
@@ -49,7 +49,7 @@ pub async fn transfer_back(ctx: &TestContext) -> Result<(), Failed> {
     }
     // wait until alice sees the transaction
     timeout(
-        ctx.config.sync_timeout(),
+        ctx.config.sync_timeout().saturating_mul(2),
```

Comment:
saturating shouldn't matter here, but can we please use checked instead? :)

Reply:
Done